arrows/colmap/colmap.cxx

```diff
@@ -0,0 +1,65 @@
+#include <arrows/colmap/colmap.h>
```

Comment:
Let's rename this file `image_container.cxx` and the header file `image_container.h` to be consistent with the other arrows.